packages/grafana-ui/src/components/Forms/InlineSegmentGroup.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC } from 'react';
+import { cx, css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { useTheme } from '../../themes';
+
+export interface Props {
+  grow?: boolean;
+  className?: string;
+}
+
+export const InlineSegmentGroup: FC<Props> = ({ children, className, grow, ...htmlProps }) => {
```

Comment:
The component looks like `InlineFieldRow` but with a bit different styling. Could we maybe use `InlineFieldRow` here and pass it the styles instead, so `InlineSegmentGroup` would be just a light wrapper around `InlineFieldRow`?

Reply:
> The name InlineSegmentGroup implies that this component should be used only for grouping segment elements, what about other input-related components?

Yea, not sure what the best name is. 
* InlineElementGroup?

> Looking at the screenshot I realized that labels and segments look exactly the same, so it's hard to distinguish which one is which, this becomes even more confusing when they wrap on multiple lines. Should we somehow make them look a bit different? 

Segments and labels are the same, all used the gf-form-label class in the old styles. 

InlineFieldRow has flex-wrap: wrap so we cannot use that, and it has no margin-bottom. Using InlineFieldRow and overriding styles seems like a bad option as this is a very common pattern (to have list of labels and no inputs, some needs to be grouped together to wrap together and that is where we need this component) 

It is more similar to InlineField than InlineFieldRow, but InlineField has margin-right and requires a label + child which makes it impossible to use for label groups that might be only one label or sometimes 2-3 labels and no input and with labels have the right margin built-in makes it impossible to use InlineField with no child or with InlineLabel as a child (it will create double margin right)

Reply:
> Yea, not sure what the best name is.
InlineElementGroup?

Maybe just `InlineGroup`?

Reply:
wonder if we should rename InlineFieldRow to InlineFormRow ?

Reply:
Think InlineGroup is maybe too unclear, InlineSegmentGroup feels more clear that it's a group of inline form segment elements (labels or inputs).

Reply:
> wonder if we should rename InlineFieldRow to InlineFormRow ?

To me it suggests that we're using it inside a form, which we don't do for inline elements.